backend/tests/test_profile.py

```
@@ -0,0 +1,167 @@
+        db.close()
+
+app.dependency_overrides[get_db] = override_get_db
+
+# Initialize FastAPI test client
+client = TestClient(app)
+
+# ----------------------
+# 🔧 Fixture for setup and teardown
+# ----------------------
+@pytest.fixture(autouse=True)
+def setup_and_teardown():
```

Ruff (0.8.2)

Missing return type annotation for public function `setup_and_teardown`

(ANN201)